src/js/view/view.js

```diff
@@ -810,9 +840,30 @@ function View(_api, _model) {
         _model.set('intersectionRatio', entry.intersectionRatio);
     };
 
+    this.setFloatingIntersection = function (entry) {
+        console.log(entry.intersectionRatio);
```

Comment:
This method needs some improvement - apart from the `console.log`, I feel we can optimize the listeners below.